### PR TITLE
PR for three outputs

### PR DESCRIPTION
**Pull request by fmustapha**
#### What does this PR do?
It shows outputs for;
-  Checking repository health
-  Setting up your repository
-  Using design patterns

#### Description of Task to be completed?
- A screenshot of your code coverage and GPA badges in your repo's README file
- The repository for your proposed project and at least one Pull Request
- A Pull Request showing two Design Patterns (suggestions: Decorator, Factory, Observer, Prototype...) or a detailed explanation of Design Patterns used while solving-the-problem

#### How should this be manually tested?

- For `checking repository health` output: Visit the [link](https://github.com/andela-fmustapha/inverted-index) and view README.md file at the end of the page
- For `setting up your repository` output: Visit the [link](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo-kakashi-fmustapha/06-building-a-program-start-to-finish/setting-up-your-repository/output)
- For `Using Design Patterns` output: Visit [link](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo-kakashi-fmustapha/06-building-a-program-start-to-finish/using-design-patterns/output)

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

N/A

#### Screenshots (if appropriate)
N/A

#### Questions
N/A

### CODE REVIEW

File: 06-building-a-program-start-to-finish/using-design-patterns/output/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  createIndex(book, filename) {
+
+  }
+
+```
+To use the methods of the object you first have to create an instance of the class by doing: 
+`invertedIndex = new InvertedIndex()`
+
+#### AUTHOR
+- **Faith Mustapha**
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/using-design-patterns/output/README.md:40]: Looks good!

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+### Link to GitHub Repo
+
+- [My InvertedIndex Repo](https://github.com/andela-fmustapha/inverted-index/ "Visit Faith's Project Repo")
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:8]: Did you see the ReadMe guidelines?

I think you should ensure it conforms to that baseline so it's easy to understand what the repo is about.

File: 06-building-a-program-start-to-finish/checking-repository-health/output/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Checking Repository Health
+----
+#### Output
+
+- **Using**: `Code Climate` and `Coveralls`
+- **Deliver**: a screenshot of your code coverage and GPA badges in your repo's README file
+
+##### README 
+- Readme screenshot 
+- ![badge screenshot](https://cloud.githubusercontent.com/assets/26274319/24551993/a8ee93d2-161c-11e7-835f-62d3ee3bafdc.png)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/checking-repository-health/output/README.md:10]: When do we expect these badges to be green?

File: 06-building-a-program-start-to-finish/using-design-patterns/output/README.md (hunk 1 above)
Comment (fmustapha) [reply 06-building-a-program-start-to-finish/using-design-patterns/output/README.md:40]: Thank you.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md (hunk 2 above)
Comment (fmustapha) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:8]: Okay I will, thank you.

File: 06-building-a-program-start-to-finish/checking-repository-health/output/README.md (hunk 3 above)
Comment (fmustapha) [reply 06-building-a-program-start-to-finish/checking-repository-health/output/README.md:10]: Very soon.

**Pull request closed without merging**